velox/dwio/parquet/tests/ParquetWriterTestBase.h

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <gtest/gtest.h>
+#include <string>
+#include "velox/common/base/Fs.h"
+#include "velox/dwio/common/FileSink.h"
+#include "velox/dwio/common/tests/utils/BatchMaker.h"
+#include "velox/dwio/common/tests/utils/DataFiles.h"
+#include "velox/dwio/parquet/writer/Writer.h"
+#include "velox/exec/tests/utils/TempDirectoryPath.h"
+#include "velox/vector/tests/utils/VectorMaker.h"
+
+namespace facebook::velox::parquet {
```

Comment:
Should this be `facebook::velox::dwio::parquet`?
The `ParquetReaderTestBase.h` header uses that namespace.

Reply:
I follow the convention that uses namespace `facebook::velox::dwio::parquet` in `velox/dwio/parquet` and use namespace `facebook::velox::dwio::dwrf` in `velox/dwio/dwrf`.

Reply:
>ParquetReaderTestBase.h header uses that namespace

I found the namespace used may be kind of inconsistent in `velox/dwio/parquet/test` and used `using namespace` in the header file. I've fixed the `using namespace` in the header file in https://github.com/facebookincubator/velox/pull/6719. I will unify the namespace in the`ParquetTpchTestBase` and `ParquetReaderTestBase` to `facebook::velox::parquet` once these pr are landed.  Do you think this is correct? @majetideepak @czentgr

Reply:
I think the convention is to follow the directory structure `facebook::velox::dwio::parquet`. `dwrf` was probably moved to `dwio` later and the namespace was likely not fixed.

Reply:
@duanmeng, You were right before. I just looked at other Parquet files inside `dwio/parquet` and all of them follow `facebook::velox::parquet`. Can you revert your last commit? Let's fix `ParquetReaderTestBase ` as you suggested separately.

Reply:
@majetideepak Sure, done.